console/src/main/java/pl/isa/freshmenindustries/user/UserManager.java

```diff
@@ -0,0 +1,4 @@
+package pl.isa.freshmenindustries.user;
+
+public class UserManager {
```

Comment:
To ma być pusta klasa?

Reply:
To nie wiem skąd się wzięło, jeszcze consolowa zaległość :) Usunięte.